raft/src/test/java/org/apache/kafka/raft/internals/KafkaRaftMetricsTest.java

```diff
@@ -162,16 +218,18 @@ public void shouldRecordVoterQuorumState(KRaftVersion kraftVersion) {
             getMetric(metrics, "current-vote-directory-id").metricValue()
         );
         assertEquals((double) 1, getMetric(metrics, "current-epoch").metricValue());
-        assertEquals((double) -1L, getMetric(metrics, "high-watermark").metricValue());
+        assertEquals((double) -1L, getMetric(metrics, "high-watermark").metricValue()); // todo, bug fix
```

Comment:
@jsancio the HW drops to -1L after candidate transitions to leader - if you agree this is a bug I'll file a Jira for this

Reply:
Okay. I think it is fair to file a bug but assign it to yourself or me.

Reply:
[KAFKA-18560](https://issues.apache.org/jira/browse/KAFKA-18560)